include/mega/request.h

```diff
@@ -82,6 +80,7 @@ class MEGA_API RequestDispatcher
     void add(Command*);
 
     bool cmdspending() const;
+    bool cmdsInflight() const;
```

Comment:
It looks like this method is declared, but not defined or used

Reply:
Ah yes I should have left that one out of this PR, it's important for the SIC removal but not used at this stage.  I'll remove it for now.